### ISSUE TITLE
Option to save ggplot graph object? 

### ISSUE THREAD

**Issue by salleuska** [nimble-dev/compareMCMCs#7]
So that one can customize the graph.

**Comment by perrydv** [comment]
@salleuska Thanks for this suggestion.  Can you clarify?  There is a ggplot object for each figure, but they are combined via html rather than into one large gpplot object.  I guess what you mean is for the user to get any of the ggplot objects directly for purposes of saving or whatever they want to do with it?  We could probably have an option to return a list of the ggplot objects.

**Comment by salleuska** [comment]
> I guess what you mean is for the user to get any of the ggplot objects directly for purposes of saving or whatever they want to do with it?

Yes, so one can customize the plot (e.g. change colors) or whatever, without having to manually get reconstruct the info inside the ggplot object.
